local-notifications/android/src/main/java/com/capacitorjs/plugins/localnotifications/LocalNotificationSchedule.java

```diff
@@ -20,8 +20,7 @@
 
     private Boolean whileIdle;
 
-    public LocalNotificationSchedule(JSObject jsonNotification) throws ParseException {
-        JSObject schedule = jsonNotification.getJSObject("schedule");
+    public LocalNotificationSchedule(JSObject schedule) throws ParseException {
         if (schedule != null) {
```

Comment:
Since you are checking if `schedule` is `null` before creating the `LocalNotificationSchedule` in `LocalNotification.java`, you can remove the `null` check from here. Other than that, changes look good.

Reply:
Yeah - I thought I took care of that, thanks for pointing it out!